app/renderer/js/components/tab.ts

```diff
@@ -28,7 +28,12 @@ export default class Tab extends BaseComponent {
 	}
 
 	activate(): void {
-		this.$el.classList.add('active');
```

Comment:
This removal breaks highlighting of the active tab.

Reply:
It breaks, but later it will handle the highlighting of the active tab. But I need to go through the code from beginning to cover all the test cases.